clients/include/hipsolver_dispatcher.hpp

```diff
@@ -120,8 +122,10 @@ class hipsolver_dispatcher
             {"sygvdx", testing_sygvdx_hegvdx<API_COMPAT, false, false, T>},
             {"sygvj", testing_sygvj_hegvj<API_NORMAL, false, false, T>},
             {"sytrd", testing_sytrd_hetrd<API_NORMAL, false, false, T>},
+#ifdef HAVE_ROCSPRASE
```

Comment:
Spelling error on ROCSPARSE.

Have a look at https://github.com/ROCm/hipSOLVER/pull/242, since it contains my approach to the same issue. I think we should have a compile definition that is set in the benchmark client CMake, since IIRC it's possible to build the client without building the library.

Reply:
Good catch. Let's get your fix merged in.